### PR TITLE
Issue 27 just update the bundle symbolic name

### PR DESCRIPTION
**Pull request by tomjenkinson**
Manifest-Version: 1.0
Bundle-License: http://www.eclipse.org/legal/epl-2.0, https://www.gnu.
 org/software/classpath/license.html
Bundle-SymbolicName: jakarta.transaction-api
Archiver-Version: Plexus Archiver
Built-By: still
Require-Bundle: system.bundle
Bnd-LastModified: 1544540660670
Implementation-Vendor-Id: org.glassfish
Bundle-DocURL: https://github.com/eclipse-ee4j
Import-Package: javax.enterprise.context,javax.enterprise.util,javax.i
 nterceptor;version="[1.2,1.2.98)",javax.transaction;version="1.3.2",j
 avax.transaction.xa
Export-Package: javax.transaction;uses:="javax.interceptor,javax.trans
 action.xa,javax.enterprise.util,javax.enterprise.context";version="1.
 3.2"
Bundle-Name: javax.transaction API
Extension-Name: javax.transaction
Bundle-Description: Java(TM) JTA 1.3 API Design Specification
Automatic-Module-Name: java.transaction
Implementation-Version: 1.3.2
Bundle-ManifestVersion: 2
Specification-Vendor: Oracle Corporation
Bundle-Vendor: EE4J Community
Tool: Bnd-0.0.255
Implementation-Vendor: EE4J Community
Bundle-Version: 1.3.2
Created-By: 1.8.0_144 (Oracle Corporation)
Build-Jdk: 1.8.0_144
Specification-Version: 1.3

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -147,7 +147,7 @@
                     </supportedProjectTypes>
                     <instructions>
                         <Bundle-Version>${spec.bundle.version}</Bundle-Version>
-                        <Bundle-SymbolicName>${spec.bundle.symbolic-name}</Bundle-SymbolicName>
+                        <Bundle-SymbolicName>jakarta.transaction-api</Bundle-SymbolicName>
```
Comment (Tomas-Kraus) [inline pom.xml]: You can keep ${spec.bundle.symbolic-name} property here with spec-version-maven-plugin 2.0 - it returns proper value.

**Comment by anajosep** [comment]
> 
> 
> @anajosep - please can you run TCK on https://jenkins.eclipse.org/jta/job/eclipse-ee4j_jta-api-pulls/60/artifact/glassfish/appserver/distributions/glassfish/target/glassfish.zip
@tomjenkinson 
The Test results are available here:
https://jenkins.eclipse.org/jakartaee-tck/job/jakartaeetck-certification/32/junit-reports-with-handlebars/testSuitesOverview.html

The JTA suite is clean. The sigtest failures are because this GF bundle did not have jakarta prefixed jars. Probably it was not build with the latest from GF EE4J_8 branch.

**Comment by anajosep** [comment]
> 
> 
> @anajosep this is how I am building it, do you think I am doing something wrong:
> 
> ```
> git clone https://github.com/eclipse-ee4j/glassfish.git
> cd glassfish
> git checkout -b EE4J_8
> mvn -Pstaging clean install -Djavax.transaction-api.version=$SNAPSHOT_VERSION
> ```

Since we are trying to checkout an existing branch, I think you should be using the command as
`git checkout EE4J_8`. (without the -b option)
If you use the option `-b` in the `git checkout` command, I think it would create a new local branch from the latest master. That could be a reason why the jakarta prefixed jars were missing in the bundle.

**Comment by anajosep** [comment]
> @anajosep of course - sorry! Would you like me to provide a different zip or do you think it is safe as it includes my JTA API?
I think another run is not required, since the JTA part got tested with the CTS run and it was clean. Please go ahead with the merge.

**Pull request merged**